build/templates/python_types.py

```diff
@@ -64,10 +64,16 @@ def python_type(self):
 class ViBoolean(int):
     @classmethod
     def from_param(cls, param):
-        return 1 if bool(param) else 0
+        return True if bool(param) else False
 
     def python_type(self):
-        return 'integer'
+        return 'bool'
+
+    def __new__(cls, val=0):
+        if val:
```

Comment:
You should rewrite this in one line like in 67, it's odd to use two styles within a few lines of code.